packages/cozy-harvest-lib/src/components/Datacards/index.jsx

```diff
@@ -24,6 +26,7 @@ const Datacards = ({ konnector, account, trigger }) => {
           konnector={konnector}
           trigger={trigger}
           accountId={trigger.message.account}
+          sourceAccountIdentifier={models.account.getAccountLogin(account)}
```

Comment:
why don't we use `getIdentifier` from `models.manifest`? In order to have the full implementation? Instead of relying of something that is not future proof?

Reply:
Also I'm wondering if you should not remove this `sourceAccountIdentifier` and just pass the `account`. Also if we do that, we can remove the `accountId` props. 

I think it's way better to pass the full object instead of a subset. The subset is never future proof.

Reply:
getIdentifier only returns which field in the manifest is the identifier, it does not return the value of the identifier field, which is the sourceAccountIdentifier only for node konnectors.

But actually, I see that getAccountLogin (which is used by getAccountName as a second possibility of value) does not use getIdentifier and I will fix that.

For the second message on passing the account, that was actually my first guess but I thought each card would have to guess how to find the sourceAccountIdentifier and I found it safer to find the sourceAccountIdentifier in only one place. But since every card is defined in only one place, I suppose we can also do it there.

Reply:
I think also getIdentifier should be renamed to getIdentifierField to avoid the confusion

Reply:
https://github.com/cozy/cozy-client/pull/1439

Reply:
can you update the peerDep and the devDep of cozy-client in order to get  https://github.com/cozy/cozy-client/pull/1439 ?

Reply:
@doubleface Too bad to have merged it without the upgrade of cozy-client 🤔